docs/getting-started/clients/web-vault/index.mdx

```diff
@@ -54,6 +54,15 @@ mkcert -cert-file dev-server.local.pem -key-file dev-server.local.pem localhost
 
 2.  Open your browser and navigate to `https://localhost:8080`.
 
+3. If your client and server are running you should see the Bitwarden login screen. Lets create an account and verify it.
+  
+    - Create a new account. This could be a fake email
+    - Log into the new account.
+    - Click on Verify Email Address
+    - Navigate to `http://localhost:1080/`
```

Comment:
I believe this is only valid if you are running the web vault against a locally running server and have set up MailCatcher; that may be a good detail to have somewhere in this bullet point as well 🙂

Reply:
Updated. Thanks!